general-cascade/twig/srcset.html

```diff
@@ -1,30 +1,30 @@
 {% set rand = random(3) + 1 %}
```

Comment:
Since the "rand" variable was being used to assign pictures to random cdn domains can you remove this completely? Or does it get used somewhere else?

Reply:
good catch